lib/Mojolicious/Guides/Rendering.pod

```diff
@@ -42,6 +42,10 @@ All templates should be in the C<templates> directories of the application,
 which can be customized with L<Mojolicious::Renderer/"paths">, or one of the
 the C<DATA> sections from L<Mojolicious::Renderer/"classes">.
 
+Any layouts should be directly under one of the C<templates> directories of the
+application (for example C<templates/layouts/example.layout.html.ep>) or one of
+the C<DATA> sections from L<Mojolicious::Rendered/"classes">.
```

Comment:
I wasn't quite sure where to put this and put it in Rendering.pod mostly for lack of a better place.

As another project owner commented, "moving documentation to the right place is the easy part" - it's the writing that is not.

I'm wondering if anyone had a better suggested place to put something like this?